aten/src/ATen/native/TensorShape.cpp

```diff
@@ -613,6 +613,13 @@ Tensor sum_to_size(const Tensor& self, IntArrayRef size) {
   return sum_to(self, size);
 }
 
+Tensor make_qtensor(const Tensor& self, IntArrayRef size, IntArrayRef stride, QuantizerPtr quantizer) {
```

Comment:
I think you can defined a new `as_strided_tensorimpl` and remove the current `as_strided_tensorimpl`